flocker/node/agents/test/blockdevicefactory.py

```diff
@@ -54,6 +52,7 @@ class ProviderType(Names):
     """
     openstack = NamedConstant()
     aws = NamedConstant()
+    rackspace = NamedConstant()
```

Comment:
`rackspace` is here as a convenience to the user, and for compatibility with the acceptance testing format, which knows explicitly about rackspace.